python/CodeEditor.py

```diff
@@ -654,8 +654,8 @@ def RejectChange(index):
     log.debug("RejectChange")
     group = g_groups[g_change_index]
     log.debug(group)
-    start_line = group.get('start_line', 1) + g_restored_lines
-    end_line = group.get('end_line', 1) + g_restored_lines
+    start_line = group.get('start_line', 1) - g_restored_lines
+    end_line = group.get('end_line', 1) - g_restored_lines
```

Comment:
I don't think this is correct. Restored lines are "deleted lines" that were restored by rejecting this change.
We need to add this new lines to compensate the changed line numbers after this change, so that the next change is proceeded at the correct start_line.
When lines are added (restored) the start_line of the next change must increase.